designs/developer/add-45/Pipelines-update/Pipelines-update.md

```diff
@@ -0,0 +1,53 @@
+---
+parent: Developer
+version: 4.5
+---
+# Add page- Updates
```

Comment:
Rather than creating a new page with the Updates that we are making for 4.5 should this just update the current documentation instead? Probably something @serenamarie125 would want to make the call on depending on the goal of the docs :)

Reply:
We'll also need to include a link to this design within the 4.5 section of the Developer page located at `/designs/developer/index.md`. Currently the full URL would be `https://openshift.github.io/openshift-origin-design/designs/developer/add-45/Pipelines-update/Pipelines-update`.

Reply:
This page Laura created (Pipelines-update) is a update to a page from 4.3 that had the path https://github.com/openshift/openshift-origin-design/blob/master/designs/developer/add-43/sections/pipelines/pipelines.md. 

This design moved some components around in the flow so it did change the documentation considerably. Laura and I talked earlier about how to make this change. So definitely looking for suggestions! 

Laura, I think we should look at the wording too. Right now you refer to this as an "Update". But I'm concerned that this may be confusing for long term documentation. So we may want to remove that kind of terminology since it is evident from the release number at the top.

Reply:
Updating the title to "Pipelines section in Add Flows (Updates in 4.5)" makes sense to me. After talking earlier this week @serenamarie125  @beaumorley  we agreed that adding a new 4.5 section and putting it there was the right location correct?